CITATION.cff

```diff
@@ -0,0 +1,29 @@
+cff-version: 1.2.0
+title: SynTest-Framework
```

Comment:
Should this not be SynTest-Core ?

Reply:
Summary of an offline conversion:
The name core is not very suitable anymore as the core now contains not only common libraries for the other repositories but also the new modular extendable feature set. This allows people to build tools like our other repos on top off the framework.

Currently, it is also confusing what the framework actually refers to: (1) all repos combined, or (2) the modular extendable ecosystem contained in this repo. We think it should be the second, therefore, we have decided to rename the repo to framework.